### PR TITLE
Added missing params to ManagementClient.getDeviceCredentials

### PR DESCRIPTION
**Pull request by zen-crab**
The type definition for `ManagementClient.getDeviceCredentials` was missing the `params` parameter. 
I added an `interface` definition for the `params` parameter and the device credentials response.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
[Auth0 Node.js client library API](https://auth0.github.io/node-auth0/ManagementClient.html#getDeviceCredentials)
[Auth0 Management API - Device Credentials](https://auth0.com/docs/api/management/v2#!/Device_Credentials/get_device_credentials)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by sbmelvin** [comment]
My last commit added the `last_used` field to `DeviceCredentials`. It's not listed in the [Management API docs](https://auth0.com/docs/api/management/v2#!/Device_Credentials/get_device_credentials), however it is a field Auth0 includes in the response:
Screen Shot 2022-03-28 at 1 09 41 PM

File: types/auth0/index.d.ts [hunk 1]
```diff
@@ -1312,6 +1312,27 @@ export interface VerifyEmail {
     } | undefined;
 }
 
+export interface GetDeviceCredentialsParams {
+    user_id: string;
```
Comment (peterblazejewicz) [inline types/auth0/index.d.ts:1316]: is any of those params required? like `user_id`?

Comment (sbmelvin) [reply types/auth0/index.d.ts:1316]: The auth0 module will accept anything, however the `user_id` field is required by the `/api/v2/device-credentials` endpoint. The endpoint's documentation says it 'Retrieves device credentials for a given user_id'. Using `getDeviceCredentials` without `user_id` results in this response: _Query validation error: 'Object didn't pass validation for format user-id: ' on property user_id (user_id of the devices to retrieve)._

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@sbmelvin thx!

**Comment by sbmelvin** [comment]
Ready to merge

**Pull request merged**